docker/docker-compose.yml

```diff
@@ -1,35 +1,65 @@
 version: '3'
 services:
+  celery:
+    image: celery-bottlecrm-broker
+    build:
+      context: ..
+      dockerfile: docker/celery.dockerfile
+    container_name: celery-bottlecrm
+    depends_on:
+      - crm-app
+      - redis
+    networks:
+      - custom_network
+
   crm-app:
     image: djcrm:1
+    build:
+      context: ..
+      dockerfile: docker/dockerfile
     container_name: crm-app
     environment:
       - DBNAME=crm
       - DBUSER=crm
       - DBPASSWORD=crm
       - DBHOST=crm-db
       - DBPORT=5432
-
     depends_on:
       - crm-db
+      - redis
+    ports:
+      - "8000:8000"
+    networks:
+      - custom_network
+
+  redis:
+    image: redis:latest
+    container_name: redis-bottlecrm
     ports:
-      - 8000:8000
+      - "6379:6379"
     networks:
-      - nw
+      - custom_network
 
   crm-db:
     image: postgres:latest
     container_name: crm-db
+    volumes:
+      - /var/lib/containers/storage/volumes/bottlecrm/_data:/var/lib/postgresql/data/pgdata
```

Comment:
this is left as a reference only to providing volume that i created using podman for local mount which ofcourse could be modified to a more standard mount point.